src/silo/query_engine/actions/insertions.cpp

```diff
@@ -220,8 +221,16 @@ void InsertionAggregation<SymbolType>::addAggregatedInsertionsToInsertionCounts(
       const std::map<std::string, std::optional<std::variant<std::string, int32_t, double>>> fields{
          {std::string(POSITION_FIELD_NAME),
           static_cast<int32_t>(position_and_insertion.position_idx)},
+         {std::string(INSERTED_SYMBOLS_FIELD_NAME),
+          std::string(position_and_insertion.insertion_value)},
          {std::string(SEQUENCE_FIELD_NAME), sequence_name},
-         {std::string(INSERTION_FIELD_NAME), std::string(position_and_insertion.insertion_value)},
+         {std::string(INSERTION_FIELD_NAME),
+          fmt::format(
+             "ins_{}:{}:{}",
```

Comment:
In LAPIS, we had the requirement that we don't show the sequence name, if there is only one segment:

https://github.com/GenSpectrum/LAPIS/blob/86d85c6137e5a6b06776829bbae254c213f8f065/lapis2/src/main/kotlin/org/genspectrum/lapis/model/SiloQueryModel.kt#L112-L119